### PR TITLE
Codeowners

### PR DESCRIPTION
**Pull request by ermirizio**
Update/Sync Releng templates for tyk

:robot: Beep, boop, this PR has been generated by [automation](https://github.com/TykTechnologies/gromit/blob/master/cmd/policy.go) on Wed Dec 14 19:54:02 UTC 2022

This PR updates the release engineering specific code in the tyk repo,
and has been generated because some releng code was changed, and
the corresponding templates were updated.

These changes will mostly be limited to the `ci/` directory and `.github` directory.

If you think this was made by mistake, or if merging this might cause any instabilities,
please log a jira ticket in the TD project citing this PR.

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4535/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3698247910)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ci/* @TykTechnologies/devops
```
Comment (asutosh) [inline .github/CODEOWNERS:1]: Should this be `/ci/ @TykTechnologies/devops` ?

Ref:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

```

> # In this example, @doctocat owns any file in the `/docs`  
> # directory in the root of your repository and any of its  
> # subdirectories.  
> docs/ @doctocat

```

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4535/merge
Commit: acb0c5985ef6a2641306c6b621d29abc59ba5e77 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3702400126)

File: .github/CODEOWNERS (hunk 1 above)
Comment (ermirizio) [reply .github/CODEOWNERS]: Good catch @asutosh , thanks!!!

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4535/merge
Commit: cdbd275bcc13efe2dec885664234d4bb8987805e 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3703209937)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4535/merge
Commit: 70fbd4d6ee43641e34a6b6220c30962608a5a7ce 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3704881064)

**Review by asutosh — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4535/merge
Commit: 8873d58219971d5d49f878062d586ac48435a0c9 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3734555064)

**Review by alephnull — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: a4b89872c01e4f5411c41b0b6cc77b288746535f Codeowners (#4535)

Update/Sync Releng templates for tyk


:robot: Beep, boop, this PR has been generated by
[automation](https://github.com/TykTechnologies/gromit/blob/master/cmd/policy.go)
on Wed Dec 14 19:54:02 UTC 2022

This PR updates the release engineering specific code in the tyk repo,
and has been generated because some releng code was changed, and
the corresponding templates were updated.

These changes will mostly be limited to the `ci/` directory and
`.github` directory.

If you think this was made by mistake, or if merging this might cause
any instabilities,
please log a jira ticket in the TD project citing this PR.

Co-authored-by: Gromit <policy@gromit>
Co-authored-by: Alok G Singh <alok@tyk.io>
Triggered by: push (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3738613053)

**Pull request merged**